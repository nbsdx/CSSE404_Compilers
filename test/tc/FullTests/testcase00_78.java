class EntryPoint {
	public static void main(String[] args) {
		int x = 1;
		if(1) {
			x = 2;
		} else {
			x = 3;
		}
		while(1) {
			x = 4;
		}
	}
}