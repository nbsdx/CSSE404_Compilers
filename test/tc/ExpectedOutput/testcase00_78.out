Condition for if statement is of type int instead of boolean
While loop condition must be a boolean.  The expressions has type int
